cloud-batch-python/python-batch/build/lib/create_container_job.py

```diff
@@ -0,0 +1,87 @@
+#  Copyright 2022 Google LLC
```

Comment:
should this be something we _embed_ in the repo or that the user can install separately?